firefox-ios/Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -16,12 +16,12 @@ class LaunchCoordinator: BaseCoordinator,
                          ParentCoordinatorDelegate {
     private let profile: Profile
     private let isIphone: Bool
+    let windowUUID: WindowUUID
```

Comment:
why remove private? a couple of other classes also exposes this property, but I don't see it being used